configure.ac

```diff
@@ -3050,6 +3050,10 @@ then
     then
         AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_ECIES_ISO18033"
     fi
+    if test "$ENABLED_ECC_ENCRYPT" = "geniv"
```

Comment:
Can you add `geniv` to the `eccencrypt` help string?

Reply:
Fixed.